docs/manual/amdbconfig.md

```diff
@@ -30,8 +30,8 @@ amdb源码编译需要用到gradle，需要预先安装gradle。
 
 从官网下载对应版本的 gradle 安装包，解压到对应目录(请将XXX换成对应的版本号)。
 ```bash
-	mkdir /software/
-	unzip -d /software/ gradleXXX.zip
+$ mkdir /software/

```

Comment:
Please don't usr /*. This require super user, is very unfriendly

Reply:
qiangwen will fix it.